torchbox/torchbox.com
# Add favicons back into images folder

No ticket

### Description of Changes Made
Fix for error on the staging site where the favicon files are not being found - they were removed by mistake when stripping out the front-end code.

### How to Test
Visit the local build, and inspect the source code. Look for `<link rel="apple-touch-icon"...>` in the `<head>`, and the following lines. Check that the images referenced exist and don't give a 404.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability
